diff --git a/ingestion/examples/airflow/airflow.cfg b/ingestion/examples/airflow/airflow.cfg
index d723d59b..9e526f8d 100644
--- a/ingestion/examples/airflow/airflow.cfg
+++ b/ingestion/examples/airflow/airflow.cfg
@@ -1,7 +1,7 @@
 [core]
 # The folder where your airflow pipelines live, most likely a
 # subfolder in a code repository. This path must be absolute.
-dags_folder = /ingestion/examples/airflow/dags
+dags_folder = /opt/airflow/dags
 
 # Hostname by providing a path to a callable, which will resolve the hostname.
 # The format is "package.function".
@@ -416,13 +416,13 @@ access_control_allow_origin =
 [lineage]
 backend = airflow_provider_openmetadata.lineage.openmetadata.OpenMetadataLineageBackend
 airflow_service_name = local_airflow_3
-openmetadata_api_endpoint = http://localhost:8585/api
+openmetadata_api_endpoint = http://openmetadata.example.com:8585/api
 auth_provider_type = no-auth
 
 [openmetadata_airflow_apis]
-dag_runner_template = /airflow/dag_templates/dag_runner.j2
-dag_generated_configs = /airflow/dag_generated_configs
-dag_managed_operators = /airflow/dag_managed_operators
+dag_runner_template = /opt/airflow/dag_templates/dag_runner.j2
+dag_generated_configs = /opt/airflow/dag_generated_configs
+dag_managed_operators = /opt/airflow/dag_managed_operators
 
 [atlas]
 sasl_enabled = False
